docs/DeveloperGuide.adoc

```diff
@@ -588,8 +588,11 @@ Use case ends.
 [none]
 * 1a. The amount keyed in by the user is invalid.
 +
-<<wallet-invalid-amount, Steps 1a1-1a2>> of [.underline]#recording an expense (UC1)# are repeated
-until the amount keyed in by the user is valid.
+[none]
+** 1a1. Wallet shows an error message.
+** 1a2. User re-enters the expense.
++
+Steps 1a1-1a2 are repeated until the amount keyed in by the user is correct.
```

Comment:
The prof commented on our use cases last time and said that don't repeat the steps if it's mentioned on the top. These steps are mentioned in UC1, that's why I put a link to refer back to UC1.